README.md

```diff
@@ -79,6 +79,19 @@ Use this [shortcode](https://gohugo.io/content-management/shortcodes/):
 ```
 end_insert -->
 
+### Defaults
+
+You can set default values in your `params.toml` or equivalent. For example:
+
+```toml
+[asciinema]
+  [asciinema.defaults]
+    theme = "solarized-dark"
+    loop = true
+    autoPlay = true
+    speed = "2.5"
```

Comment:
```suggestion
[params.asciinema]
  [params.asciinema.defaults]
    theme = "solarized-dark"
    loop = true
    autoPlay = true
    speed = "2.5"
```

I had to add `params.` to get it to work locally for me in my toml.

Reply:
Was that your experience as well?

Reply:
@cljoly No, not my experience. In which file did you put the toml snippet?

I inserted this snippet into `/config/_default/params.toml`:

```toml
[asciinema]
  [asciinema.defaults]
    theme = "solarized-dark"
    loop = true
    autoPlay = true
    speed = "5.0"
```

My suspicion is that you placed your toml snippet into `/config/_default/config.toml`?

My intuition is that `params.toml` is the correct location for this, but I have no strong feelings on it.

Thanks for the great work btw.

Reply:
> My suspicion is that you placed your toml snippet into /config/_default/config.toml?

True.

> My intuition is that params.toml is the correct location for this, but I have no strong feelings on it.

I think it safer to assume that someone reading the README will be using out of the box Hugo (so a `config.toml` at the root, not in a `config` folder or anything) and so the README should give an example with `[params.asciinema]`, because as I just tried, `[asciinema]` alone would not work there.

I'm happy to learn that params.toml exists though, it's great and more practical as a config grows!

> Thanks for the great work btw.

No problem, happy to share with the community 😄